app/views/mountain_view/styleguide/show.html.erb

```diff
@@ -30,10 +30,10 @@
     <% end %>
   <% else %>
     <div class="mv-hint">
-      <h2><span>Hint:</span>You have Stubs but they don't follow the correct format</h2>
+      <h2><%= t('mountain_view.show.hint_stub_format_html') %></h2>
       <ul>
         <li>
-          You should write stub examples like this:
+          <%= t('mountain_view.show.tip_stub_format_html') %>:
```

Comment:
The `:` at the end of the line should go in the translation string (in `en.yml`)